posts/inside-rust/2021-02-15-shrinkmem-rustc-sprint.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "March Sprint for rustc: Shrink Memory Usage"
+author: Felix Klock
+team: The Compiler Team <https://www.rust-lang.org/governance/teams/compiler>
+---
+
+I am very excited about the compiler team's upcoming sprint, and
+I want to share that excitement with all of you.
+
+The Rust Compiler Team decided over a series of recent meetings
+([4 dec 2020][mtg-2020-12-04], [15 jan 2021][mtg-2021-01-15],
+[29 jan 2021][mtg-2021-01-29]) that our focus for 2021 will be improving the compiler's
+performance from the perspective of the new contributor experience bootstrapping
+the compiler and developing code for the compiler.
+
+The team's first sprint for 2021 will be during the first week of March, and its
+focus will be on reducing the memory footprint of the compiler during bootstrap.
+
+During our most recent planning meeting ([12 feb 2021][mtg-2021-02-12]),
+there were no new significant steering meeting proposals, so we decided to
+do some extra sprint planning during that time. We established:
+
+ * we should settle on a common set of tools for measuring memory usage (for now we expect it to be Valgrind DHAT on Linux).
```

Comment:
Are you saying that we're not going to be measuring memory usage on Windows at all or just that for Linux the preferred tool is DHAT?

Reply:
You're right, that is phrased ambiguously. I think my intent was to set current expectations w.r.t. Linux alone, without implying anything about non-Linux hosts.